### ISSUE TITLE
Zooming before starting record with low quality

### ISSUE THREAD

**Issue by radik** [react-native-community/react-native-camera#1452]
### Which implementation are you using

*RNCamera*

### Steps to reproduce
1. Create Camera component
```javascript
<Camera ref={ref=> this.camera = ref;} .../>
```
2. Use `recordAsync` to record video and pass `quality` option to it (lower than highest)
```
this.camera.recordAsync({
    quality: RNCamera.Constants.VideoQuality["360p"]
})
```

### Does it work with Expo Camera?
The same behaviour in expo

### Expected behaviour
Recording is started without any zoom.

### Actual behaviour
Preview will be kind of zoomed before starting to record.

### Environment
- **Node.js version**: v9.10.1
- **React Native version**: 0.51.0
- **React Native platform + platform version**: iOS 9.0, Android 5.0

### react-native-camera
**Version**: 1.0.3

**Comment by benjaminreid** [comment]
Are there any updates or workarounds on this?

**Comment by radik** [comment]
@benjaminreid No, issue is still there

**Comment by benjaminreid** [comment]
Ah, that’s a shame.

I thought it may have been related to this rendering issue as I’m overlay and causing a few re-renders when recording starts. https://github.com/expo/expo/issues/1057#issuecomment-398084402

But alas, I everything out but the camera view and just started recording on a `setTimeout` and the zoom still happens with seemingly no re-renders.

**Comment by cmmartin** [comment]
Such a little thing that really makes this unusable 😢

**Comment by juanjsebgarcia** [comment]
Sorry to necro this thread. But for those finding in the future I have found that the following works (minimal example)

```
import { RNCamera } from 'react-native-camera';

const VIDEO_QUALITY = RNCamera.Constants.VideoQuality['720p'];

// Preview
<RNCamera
  defaultVideoQuality={VIDEO_QUALTIY}
/>

export const VIDEO_OPTIONS = {
  quality: VIDEO_QUALITY,
};

// Async record
public takeVideo = async function () {
  const videoData = await this.camera.recordAsync(VIDEO_OPTIONS);
};
```

**Comment by MrXyfir** [comment]
This appears to still be an issue. @juanjsebgarcia's fix did not fix it for me, but passing `zoom={0.00225}` to the `RNCamera` component seems to at least match the same zoom level that's applied during recording so it's not such a jarring change.
